### PR TITLE
atk4/ui/fix/#491 auto complete not initializing with proper text

### PR DESCRIPTION
**Pull request by skondakov**
I am trying to load the AutoComplete value from its model and not the forms model. Also trying to show its selected title field.

### CODE REVIEW

**Comment by romaninsh** [comment]
@ibelar if this looks ok to you - merge.

**Comment by skondakov** [comment]
I have added a check for $this->field .. about setting to "0", I set it to this value as this seems to be the default empty value for the autocomplete. We can set it to null as well, but shouldn't leave it without changing as I have noticed that in case of multiple autocompletes - there is a bug with loading the same value all over them.

File: src/FormField/AutoComplete.php [hunk 1]
```diff
@@ -238,10 +238,18 @@ public function renderView()
...
-            $chain->dropdown('set value', $data['id'])->dropdown('set text', $data[$this->model->title_field]);
-            $this->js(true, $chain);
+        if ($this->field && $this->field->get()) {
+            $id_field = $this->id_field ?: $this->model->id_field;
+            $title_field = $this->title_field ?: $this->model->title_field;
+
+            $this->model->tryLoadBy($id_field, $this->field->get());
+
+            if (!$this->model->loaded()) {
+                $this->field->set('0');
```
Comment (romaninsh) [inline src/FormField/AutoComplete.php]: why not null?

**Comment by skondakov** [comment]
Done, it is null now

**Pull request merged**